### ISSUE TITLE
Question about order of population set when running Dtrios

### ISSUE THREAD

**Issue by hungweichen0327** [millanek/Dsuite#83]
Dear @millanek and community,

I have questions about the order of population set when running Dusite Dtrios.
If I have 3 populations (P1, P2, P3) and an outgroup population, the two orders in the population set below would have a different meaning when running Dsuite Dtrios. Because the first one indicates (P1, P2), P3), O) and the second one indicates (P2, P3) P1, O). Is that correct?
```
(1) P1 P2 P3 O
(2) P2 P3 P1 O
``` 

When I ran the `Dsuite Dtrios`, there were only 56 output results (meaning 56 combinations) when I had 8 populations with an outgroup population. But the output results would have 336 (8*7*6). 

Thus, I should use the `-t` option and give the tree file providing the relationship of 8 populations when running `Dsuite Dtrios`.

Thank you for the help.

**Comment by millanek** [comment]
Yes, if the tree file specifies  (P1, P2), P3), O) then that's what the _tree.txt output is going to contain. However, do double check that this is the "correct" tree -- if the _BBAA.txt output is giving you a different orientation, it is for a reason.

The meaning of the different output files "_tree.txt", "_BBAA.txt", and "_Dmin.txt" is explained the documentation here in GitHub and more in detail in the paper.

All the best
Milan
